src/docs/integrations/slack/index.mdx

```diff
@@ -2,7 +2,7 @@
 title: "Slack Integration"
 ---
 
-Based off of this [post](https://forum.sentry.io/t/how-to-configure-slack-in-your-on-prem-sentry/3463)
+Updated version of this [post](https://forum.sentry.io/t/how-to-configure-slack-in-your-on-prem-sentry/3463)
```

Comment:
Not sure if we need to reference back to the forum now as it is not relevant anymore.